src/GPEmulator.jl

```diff
@@ -117,12 +117,12 @@ function GPObj(inputs, data, package::GPJL; GPkernel::Union{K, KPy, Nothing}=not
         GPkernel_i = deepcopy(GPkernel)
         # inputs: N_samples x N_parameters
         # data: N_samples x N_data
-        logstd_obs_noise = log(sqrt(0.5)) # log standard dev of obs noise
+        logstd_obs_noise = log(sqrt(0.01)) # log standard dev of obs noise
         # Zero mean function
         kmean = MeanZero()
         m = GPE(inputs', dropdims(data[:, i]', dims=1), kmean, GPkernel_i, 
                 logstd_obs_noise)
-        optimize!(m, noise=false)
+        optimize!(m, noise=true)
```

Comment:
FYI this is the default of `optimize!`. I will try to read the documentation of GaussianProcesses.jl to see what would be the best setting for our purposes.